web/simulator/simulator.css

```
@@ -172,66 +221,73 @@ button[aria-pressed="true"] {
-  color: #d8d0c8;
-  font-family: ui-monospace, "SFMono-Regular", Consolas, "Liberation Mono", monospace;
-  font-size: 0.78rem;
+  border: 1px solid #33383d;
+  border-radius: 6px;
+  background: #101214;
+  color: #cbd2d8;
+  font-family: ui-monospace, 'SFMono-Regular', Consolas, 'Liberation Mono', monospace;
+  font-size: 0.72rem;
   line-height: 1.45;
   white-space: pre-wrap;
   word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)